README.md

```diff
@@ -205,9 +205,42 @@ packages on the destination side can be ignored with
 
 By default, all subpackages of the specified packages are also ignored, but
 this can be disabled on an individual basis by adding
-`<ignoreSubpackages>false</ignoreSubpackages>` to the `<ignoreSourcePackage>`
+`<filterSubpackages>false</filterSubpackages>` to the `<ignoreSourcePackage>`
 or `<ignoreDestinationPackage>` element.
 
+### Target only conflicts from certain packages
+
+Conversely, found conflicts can be filtered based on the package name of the class that has the
```

Comment:
a small suggestion that I think might make this read easier: 

```suggestion
Conversely, the plugin can be configured to _only_ report on conflicts in specific packages, based on the name of the class that has the
```

(I think this messes up the word wrapping though)
```